packages/hdwallet-native/src/native.ts

```diff
@@ -0,0 +1,184 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { mnemonicToSeed } from "bip39";
+import { fromSeed } from "bip32";
+import { isObject } from "lodash";
+import { getNetwork } from "./networks";
+import { MixinNativeBTCWallet, MixinNativeBTCWalletInfo } from "./bitcoin";
+import { MixinNativeETHWalletInfo, MixinNativeETHWallet } from "./ethereum";
+
+class NativeHDWalletInfo
+  extends MixinNativeBTCWalletInfo(MixinNativeETHWalletInfo(class Base {}))
+  implements core.HDWalletInfo {
+  _supportsBTCInfo: boolean = true;
+  _supportsETHInfo: boolean = true;
+  _supportsCosmosInfo: boolean = false;
+  _supportsBinanceInfo: boolean = false;
+  _supportsRippleInfo: boolean = false;
+  _supportsEosInfo: boolean = false;
+
+  getVendor(): string {
+    return "Native";
+  }
+
+  hasOnDevicePinEntry(): boolean {
+    return false;
+  }
+
+  hasOnDevicePassphrase(): boolean {
+    return false;
+  }
+
+  hasOnDeviceDisplay(): boolean {
+    return false;
+  }
+
+  hasOnDeviceRecovery(): boolean {
+    return false;
+  }
+
+  hasNativeShapeShift(): boolean {
+    return false;
+  }
+
+  describePath(msg: core.DescribePath): core.PathDescription {
+    switch (msg.coin) {
+      case "Bitcoin":
```

Comment:
We need to add `BitcoinCash` here as well. I also changed the others to lowercase and added `toLowerCase()` so we don't have to remember casing and we aren't using enums atm